update MOS documentation for 2.2.x release

stevesims opened the pull request:
Documents new MOS star commands, behaviour and APIs

stevesims committed:
update MOS documentation for 2.2.x release

Documents new MOS star commands, behaviour and APIs

astralaster commented on MOS-API.md:
```diff
@@ -109,249 +113,260 @@ See mos_api.asm for implementation
-- HL(U): Address of filename1 (zero terminated)
-- D(E)U: Address of filename2 (zero terminated)
+- `HL(U)`: Address of filename1 (zero terminated)
+-` D(E)U`: Address of filename2 (zero terminated)
```
Looks like a typo

stevesims replied:
Yep - well spotted - probably a mistake in the original docs

I’ll fix

also need to document the hotkey command.

astralaster reviewed (approved):
Looks very good, just found a minor typo.

stevesims committed:
Add in missing hotkey command

and fill in some more detail on what registers the MOS API preserves between calls
